fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayBanner.kt

```diff
@@ -198,20 +208,103 @@ private fun NormalTabsTabIcon(normalTabCount: Int) {
     }
 }
 
+/**
+ * Banner displayed in multi select mode.
+ *
+ * @param selectedTabCount Number of selected tabs.
+ * @param shouldShowInactiveButton Whether or not to show the inactive tabs menu item.
+ */
+@Suppress("LongMethod")
 @Composable
-private fun MultiSelectBanner() {
-    Box(
+private fun MultiSelectBanner(
+    selectedTabCount: Int,
+    shouldShowInactiveButton: Boolean,
+) {
+    var showMenu by remember { mutableStateOf(false) }
+    val menuItems = mutableListOf(
+        MenuItem(
+            title = stringResource(R.string.tab_tray_multiselect_menu_item_bookmark),
+        ) {},
+        MenuItem(
+            title = stringResource(R.string.tab_tray_multiselect_menu_item_close),
+        ) {},
+    )
+    if (shouldShowInactiveButton) {
+        menuItems.add(
+            MenuItem(
+                title = stringResource(R.string.inactive_tabs_menu_item),
+            ) {},
+        )
+    }
+
+    Row(
         modifier = Modifier
             .fillMaxWidth()
-            .height(80.dp)
+            .height(88.dp)
             .background(color = FirefoxTheme.colors.layerAccent),
-        contentAlignment = Alignment.Center,
     ) {
+        IconButton(
+            onClick = {},
+            modifier = Modifier.align(Alignment.CenterVertically),
+        ) {
+            Icon(
+                painter = painterResource(id = R.drawable.ic_close),
+                contentDescription = stringResource(id = R.string.tab_tray_close_multiselect_content_description),
+                tint = FirefoxTheme.colors.iconOnColor,
+            )
+        }
         Text(
-            text = "Multi selection mode",
-            color = FirefoxTheme.colors.textOnColorPrimary,
+            text = stringResource(R.string.tab_tray_multi_select_title, selectedTabCount),
+            modifier = Modifier.align(Alignment.CenterVertically),
             style = FirefoxTheme.typography.body1,
+            color = FirefoxTheme.colors.textOnColorPrimary,
+            fontSize = 20.sp,
+            fontWeight = FontWeight.W500,
         )
+
+        Spacer(modifier = Modifier.weight(1.0f))
+
+        IconButton(
+            onClick = {},
+            modifier = Modifier.align(Alignment.CenterVertically),
```

Comment:
these modifiers can be replaced by adding `verticalAlignment = Alignment.CenterVertically,` to the `Row`